### PR TITLE
Add hyperlinks to README

### PR DESCRIPTION
**Pull request by matt-eric**
I hyperlinked all Web Audio API references to their respective URLs in the [docs](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API) and made some additional miscellaneous revisions to the description text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,62 +2,61 @@
...
 
 ## Audio context states
-The [audiocontext-states](https://github.com/mdn/webaudio-examples/tree/master/audiocontext-states) directory contains a simple demo of the new Web Audio API <code>AudioContext</code> states, including the states property and the <code>close()</code>, <code>resume()</code> and <code>suspend()</code> methods. See [https://developer.mozilla.org/en-US/docs/Web/API/AudioContext](https://developer.mozilla.org/en-US/docs/Web/API/AudioContext) for more details. [Run the demo live](http://mdn.github.io/webaudio-examples/audiocontext-states/).
+The [audiocontext-states](https://github.com/mdn/webaudio-examples/tree/master/audiocontext-states) directory contains a simple demo of the new Web Audio API <code>[AudioContext](https://developer.mozilla.org/en-US/docs/Web/API/AudioContext)</code> methods, including the states property and the <code>close()</code>, <code>resume()</code> and <code>suspend()</code> methods. [Run the demo live](http://mdn.github.io/webaudio-examples/audiocontext-states/).
```
Comment (chrisdavidmills) [inline README.md:17]: "states property" — put "states" in code font?

Comment (chrisdavidmills) [reply README.md:17]: Also put a comma after "<code>resume()</code>"

File: README.md [hunk 2]
```diff
@@ -2,62 +2,61 @@
...
 
 ## Spacialization
-The [spacialization](https://github.com/mdn/webaudio-examples/tree/master/spacialization) directory contains an example of how the various properties of a <code>[PannerNode](https://developer.mozilla.org/en-US/docs/Web/API/PannerNode)</code> object can be adjusted to emulate a three-dimensional environment. For more information see [Web audio spatialization basics](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API/Web_audio_spatialization_basics). Try the [live demo](http://mdn.github.io/webaudio-examples/spacialization/).
+The [spacialization](https://github.com/mdn/webaudio-examples/tree/master/spacialization) directory contains an example of how the various properties of a <code>[PannerNode](https://developer.mozilla.org/en-US/docs/Web/API/PannerNode)</code> interface can be adjusted to emulate three-dimensional space. For more information see [Web audio spatialization basics](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API/Web_audio_spatialization_basics). Try the [live demo](http://mdn.github.io/webaudio-examples/spacialization/).
```
Comment (chrisdavidmills) [inline README.md:53]: "emulate three-dimensional space" -> "emulate sound in a three-dimensional space" ?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Thanks @matt-eric , looking mostly good. Just a few nitpicks for you to consider.

**Comment by matt-eric** [comment]
Good suggestions, @chrisdavidmills. Thanks for noticing that.

**Review by chrisdavidmills — APPROVED** [review]
Perfect! r+

**Pull request merged**